### DISCUSSION TITLE
Horizon Zero Dawn : Neither XInput nor DS4Windows is working

### DISCUSSION THREAD

**Discussion by fraf** [nefarius/DsHidMini#145] [Q&A] [answered]
Hello,

I installed dshidmini_v2.0.248.0, latest version at this time, and I did several tests on modern pc games and I report that Horizon Zero Dawn doesn't not recognized the DS3 across XInput or DS4Windows emulation.
With a native Xbox One game controller, the gamepad is directly working under menus.
I don't know if other recent games like it have this problem too.

**Comment by nefarius** [comment]
![image](https://user-images.githubusercontent.com/286631/136658094-037c829d-2ff6-42d9-90d0-b427fb43cd5e.png)

**Comment by Kanuan** [comment, marked as answer]
1. Regarding the `DS4Windows` HID Device Mode not working, are you aware that it requires the DS4Windows companion app?
    - Check the following guide: [DS4Windows Mode User Guide](https://vigem.org/projects/DsHidMini/DS4-Mode-User-Guide/)
1. Regarding the `Xinput` HID Device Mode not working, maybe it's related to the following issue: [Game compatibility issues with XInput HID Device Mode](https://github.com/ViGEm/DsHidMini/discussions/114)
    - For now. the work around to the XInput mode issues is to launch games **through** Steam (even non-Steam games) and have the 2 options shown in the image below enabled
    - ![image](https://user-images.githubusercontent.com/24910442/136658189-245ae599-abea-4a5a-a5cb-fb2e9963e455.png)

**Comment by fraf** [comment]
Alright for DS4Windows, finally working. Got it ! :)
Thanks you for this discussion.
